docs/index.rst

```diff
@@ -0,0 +1,29 @@
+.. ServiceX Client Library documentation master file, created by
+   sphinx-quickstart on Wed Oct 19 23:24:40 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to ServiceX Client Library's documentation!
+===================================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+.. toctree::
+   :hidden:
+
+   intro
+   api
+   development
+
+.. include:: ../README.md
+   :parser: myst_parser.sphinx_
+   :end-before: <!-- Comment: Index page ends here in the docs -->
```

Comment:
Generating and then _including_ rst files from Markdown is thanks to the amazing MyST project.

* https://www.sphinx-doc.org/en/master/usage/markdown.html
* https://myst-parser.readthedocs.io/en/latest/faq/index.html#include-markdown-files-into-an-rst-file